docs/src/modules/components/AppSearch.js

```diff
@@ -159,20 +159,25 @@ function DocSearcHit(props) {
   const { children, hit } = props;
 
   function displayTag(pathname) {
-    if (!pathname.match(/^\/(material-ui|joy-ui|base|x\/(react-data-grid|api))\//)) {
+    // does not need to show product label for MUI X because they are grouped by the product name in the search
+    // ie. Data Grid, Date Picker
+    if (!pathname.match(/^\/(material-ui|joy-ui|base)\//)) {
       return null;
     }
     let text = '';
+    let color = 'default';
     if (pathname.startsWith('/material-ui/')) {
       text = 'Material UI';
     }
     if (pathname.startsWith('/joy-ui/')) {
       text = 'Joy UI';
+      color = 'secondary';
     }
     if (pathname.startsWith('/base/')) {
       text = 'MUI Base';
+      color = 'success';
     }
-    return <Chip label={text} size="small" sx={{ mr: 1 }} />;
+    return <Chip label={text} size="small" variant="light" color={color} sx={{ mr: 1 }} />;
```

Comment:
@danilo-leal The trick I use is to manually change the variant & color here, then check the UI.

Remember to change it back 😂